.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+python: 2.7
+sudo: false
+cache:
+  directories:
+  - eggs
+env:
+  - PLONE_VERSION=4.3
+  - PLONE_VERSION=4.3 TEST_EGGS=plone.app.contenttypes
+matrix:
+  fast_finish: true
+install:
+  - python bootstrap.py
+  - bin/buildout buildout:test-eggs=$TEST_EGGS annotate
+  - bin/buildout buildout:test-eggs=$TEST_EGGS
+script:
+  - bin/code-analysis
+  - bin/test
+after_success:
+#  - bin/createcoverage
```

Comment:
A little documentation here would be nice, like `FIXME: Uncomment after implementing real tests, we just added CI infrastructure.`.

Reply:
Or simply "avoid timeout" with a `FIXME` as well.